src/python/README.md

```diff
@@ -8,7 +8,7 @@ Beta : Core behavior well-used and proven; bugs lurk off the beaten path.
 
 PREREQUISITES
 -------------
-- Python 2.7, virtualenv, pip
+- virtualenv, pip, python-dev, python3-dev
```

Comment:
I was under the impression that it'd be one or the other after a certain point. I'd be more comfortable with it just saying `python-dev` for now.

Also, I assume we're using package names? If so, s/`virtualenv`/`python-virtualenv` and s/`pip`/`python-pip`. `pip` also has a `python3-*` variant, so if we're listing Python 3 as a prereq we should probably mention `python3-pip`.